src/ol/reproj/DataTile.js

```diff
@@ -394,7 +403,6 @@ class ReprojDataTile extends DataTile {
         );
 
         releaseCanvas(context);
-        canvasPool.push(canvas);
```

Comment:
The removal of this line will likely cause problems in mobile Safari.

Reply:
Hopefully not as `releaseCanvas` is reducing the size to a single.

Reply:
Yes, but it takes a while for Safari to garbage collect that, so it is important to reuse the canvas.